packages/x-date-pickers/src/tests/describeValue/testControlledUnControlled.tsx

```diff
@@ -40,8 +41,13 @@ export const testControlledUnControlled: DescribeValueTestSuite<any, any> = (
       assertRenderedValue(newValue);
       // TODO: Clean this exception or change the clock behavior
       expect(onChange.callCount).to.equal(componentFamily === 'clock' ? 2 : 1);
-      // TODO: Support range
-      expect(onChange.lastCall.args[0]).toEqualDateTime(newValue as any);
+      if (Array.isArray(newValue)) {
```

Comment:
`type` is not available here, hence I've used another `date-range` differentiator. 🤔

Reply:
Why `type`  is not available? It should be returned by the `getOptions()`

But `Array.isArray()` has the advantage of not being sensible to typos 🤭

Reply:
> Why `type` is not available? It should be returned by the `getOptions()`

Because these tests are used in all pickers and we can't safely provide `new-pickers` type from this test, because it might be used in legacy pickers.

Maybe it would be worth experimenting type propagation from the consuming `describes.xxx.test.tsx` file 🤔 


> But `Array.isArray()` has the advantage of not being sensible to typos 🤭

Indeed, we have that benefit... 🙈 😆